Refresh.GameServer/Endpoints/Game/Handshake/AuthenticationEndpoints.cs

```diff
@@ -262,23 +262,17 @@ private static void SendPlatformNotAllowedNotification(GameDatabaseContext datab
     /// </summary>
     [GameEndpoint("goodbye", HttpMethods.Post, ContentType.Xml)]
     [MinimumRole(GameUserRole.Restricted)]
-    public Response RevokeThisToken(RequestContext context, GameDatabaseContext database, GameUser user, Token token, MatchService matchService)
+    public Response RevokeThisToken(RequestContext context, GameDatabaseContext database, Token token, MatchService matchService)
```

Comment:
Keep the reference to the user; removing it is bound to cause issues when we revoke the token in this method.

Reply:
Fixed in #79f24aab24d93bf4a03729004fd2cbf27b22203d